homeassistant/components/rova/const.py

```diff
@@ -0,0 +1,8 @@
+"""Const file for Rova."""
```

Comment:
There's no reason to introduce this module as there is only one module that uses these constants. We use a common constant module for common constants that are shared between more than one module.

Reply:
I want to introduce more changes but this is a first split up

Reply:
Ok, then please add that explanation to the PR description.